cradle-cassandra/src/test/java/com/exactpro/cradle/cassandra/integration/BaseCradleCassandraTest.java

```diff
@@ -164,6 +169,22 @@ protected TestEventToStore generateTestEvent (String scope, Instant start, long
         return batch;
     }
 
+    @AfterMethod(description = "Verify that no tombstones was created")
+    public void ensureNoTombstones() {
+        try {
+            long tombstonesEvents = storage.countTombstones(CassandraCradleHelper.KEYSPACE_NAME, TestEventEntity.TABLE_NAME);
```

Comment:
issue: The best solution will be check all tables in keyspace. 
`nodetool cfstats` show statistics for all tables (I'm not sure that it is actual statistics instead of approximate). We can call `nodetool cfstats` at the end of suite and paras statistic for the whole keyspace without changing Cradle API

Reply:
I've tried it. It doesn't give stable results.